lib/logstash/outputs/elasticsearch/buffer.rb

```diff
@@ -1,124 +0,0 @@
-require 'concurrent'
```

Comment:
This was the internal buffer used for `#receive`. In logstash 2.2+ we only call `#multi_receive` so no internal buffering is necessary!